app/ts/background/providerMessageHandlers.ts

```diff
@@ -17,7 +17,11 @@ export async function ethAccountsReply(simulator: Simulator, websiteTabConnectio
 	if (port.sender?.tab?.id === undefined) return returnValue
 
 	const signerAccountsReply = EthereumAccountsReply.parse(request.params)
-	const signerAccounts = signerAccountsReply[0]
+	if (signerAccountsReply[0].type === 'error') {
+		console.warn('Wallet returned an error!') // TODO, propagate this to the UI for user to see
```

Comment:
propagate wallet errors to UI

Reply:
Do we not have a generalized solution for this?  Feels risky to merge a PR that has errors only going to the console.

Reply:
well, before it was just in the pages console, I moved it to interceptors console where I plan to display it to user